bundles/org.openhab.core.karaf/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Should not be part of this PR.

Reply:
Yep, don't know how this ended up here :)